tests/lightning_tensor/test_serialize_tensor.py

```diff
@@ -0,0 +1,936 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for the serialization helper functions.
```

Comment:
Is it possible to move it to `test_serialize.py`?

Reply:
At the implementation beginning, I tried to keep the test on `test_serialize.py` but the logic to call the modules, `mpsNamesObs` or `exactNamesObs` is different and became hard to keep unified this test.